src/api/storage/controller.js

```diff
@@ -15,8 +17,10 @@ const MAIN_PAGE_ENTITIES_QUERIES = {
   topics: () => Topic.getAll(),
 };
 
-export const getMainPage = ({ user }, res, next) =>
-  StorageEntity.getValue(MAIN_PAGE_KEY)
+const SIDEBAR_ENTITIES_QUERIES = pick(MAIN_PAGE_ENTITIES_QUERIES, ['tags']);
```

Comment:
why not just `MAIN_PAGE_ENTITIES_QUERIES.tags`?

Reply:
`MAIN_PAGE_ENTITIES_QUERIES.tags` returns the only function associated with `tags`.
`pick()` preserves the structure: we get `{ tags: func() }`